David/Code-Review-438/src/Main.java

```diff
@@ -0,0 +1,255 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class Main {
+    Main() {
+        this.users = new ArrayList<>();
+        this.posts = new ArrayList<>();
+    }
+
+    class User {
+        String username;
+        String displayName;
+        String state;
+        ArrayList<String> friends;
+
+        User() {
+            this.username = "";
+            this.displayName = "";
+            this.state = "";
+            this.friends = new ArrayList<>();
+        }
+    }
+
+    class Post {
+        String postId;

```

Comment:
Adding getter and setter methods in this line too, would be a good option. As it helps in better encapsulation.